packages/eslint-plugin/configs/recommended-with-formatting.js

```diff
@@ -43,21 +43,30 @@ const config = {
 		'import/default': 'warn',
 		'import/named': 'warn',
 	},
+	overrides: [],
 };
 
 if ( isPackageInstalled( 'jest' ) ) {
-	config.overrides = [
-		{
-			// Unit test files and their helpers only.
-			files: [ '**/@(test|__tests__)/**/*.js', '**/?(*.)test.js' ],
-			extends: [ require.resolve( './test-unit.js' ) ],
-		},
-		{
-			// End-to-end test files and their helpers only.
-			files: [ '**/specs/**/*.js', '**/?(*.)spec.js' ],
-			extends: [ require.resolve( './test-e2e.js' ) ],
-		},
-	];
+	config.overrides.push( {
+		// Unit test files and their helpers only.
+		files: [ '**/@(test|__tests__)/**/*.js', '**/?(*.)test.js' ],
+		excludedFiles: [ 'test/e2e/**/*.js' ],
+		extends: [ require.resolve( './test-unit.js' ) ],
+	} );
+}
+
+if ( isPackageInstalled( '@playwright/test' ) ) {
```

Comment:
If that's problematic, it might be better to remove all rules for e2e tests in `@wordpress/eslint-plugin` completely and apply them on a project basis. At least we shouldn't maintain handling for two systems in the package that we publish to npm.

Reply:
Yeah this part always bothers me 😅. It think we can still expose them, just don't expose them in the `recommended` preset. Users should opt-in to `test-e2e-puppeteer` or `test-e2e-playwright` themselves.